natvis/saber.natvis

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This just looks like a placeholder for future work, because it doesn't seem to have anything to do with Saber.